server/views/partials/formErrorSummary.njk

```diff
@@ -0,0 +1,14 @@
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% set errorSummaryList = validationErrors | buildErrorSummaryList %}
+
+{% if errorSummaryList.length > 0 %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            {{ govukErrorSummary({
+                titleText: "There is a problem",
+                errorList: errorSummaryList,
```

Comment:
same title "There is a problem" even when there are multiple errors?

Reply:
Hm, this is how AM does it - I'm not sure what the standard is, but it looks like a lot of other MOJ projects use the same sentence.